core-services/ipi-deprovision/aws.sh

```
@@ -169,9 +250,20 @@ aws iam list-users --query "Users[?starts_with(UserName, 'ci-op-') && CreateDate
 done
 
 FAILED="$(find ${clusters} -name failure -printf '%H\n' | sort)"
+final_rc=0
+
 if [[ -n "${FAILED}" ]]; then
 	echo "Deprovision failed on the following clusters:"
 	xargs --max-args 1 basename <<< $FAILED
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)